README.md

```diff
@@ -1,46 +1,68 @@
+One of the core steps for setting up Moderne is publishing Moderne abstract syntax trees (ASTs) to an artifact repository such as Artifactory or Nexus. 
 
-It is designed for use on a Jenkins installation with the [Job DSL](https://plugins.jenkins.io/job-dsl) plugin installed.
+While it is possible to [manually add a plugin to each repository](https://app.gitbook.com/o/-MEp_3EtccewzekKY8mZ/s/-MhFwm0iG8BFZKPYoFkH/how-to/integrating-private-code) to accomplish this, there are a few problems with that approach:
+* It does not scale well in organizations with many repositories.
+* It requires modifying the build processes of code, which may not be desired.
+* The task which generates the ASTs takes time and memory. This, in turn, could negatively affect the building and shipping of code.
 
-Each repository that is subject to nightly ingest is listed in `repos.csv`, along with calculated information about its build tooling and language level requirements.
+A much better and safer way to handle the creation of ASTs is to use Jenkins to mass produce the artifacts and for Moderne to ingest them nightly or at set intervals. Please note, though, that this could mean your ASTs are out of date if it's been a long time since the last ingest. 
 
-## Modifications to use in your environment
+## Requirements
 
-1. Click the green "Use this template" button on this page to create a copy of this repository. This is distinct from a fork in that it can be copied into a private organization and does not have an upstream link back to this repository.
+This approach for creating ASTs at scheduled intervals is designed for use on a Jenkins installation with the [Job DSL](https://plugins.jenkins.io/job-dsl) plugin installed.
 
-    <img width="284" alt="image" src="https://user-images.githubusercontent.com/1697736/189235703-0b7c1dcd-1e73-43f1-81d9-a39c617449c4.png">
+Each repository that is subject to nightly ingest is listed in `repos.csv`, along with calculated information about its build tooling and language level requirements. The instructions for generating this file can be found [below](#specify-what-repositories-should-be-ingested).
 
-2. Change this line in [add-repos.sh](https://github.com/moderneinc/enterprise-jenkins-ingest/blob/main/add-repos.sh#L76) to point to your version control system. Note that it does not have to be a GitHub installation; any git server will work.
+## Setup Instructions
 
-The `seed.groovy` Job DSL script generates jobs that publish ASTs either to Artifactory or Nexus. They are mutually exclusive, so please follow one of the two configuration options.
+### Copy this repository
 
-### If you want to publish ASTs to Artifactory
+Click the green "Use this template" button on this page to create a copy of this repository. This is distinct from a fork in that it can be copied into a private organization and it does not have an upstream link back to this repository.
```

Comment:
In your next section you have a "if you don't use GitHub", but this section assumes they have GitHub. My suggested edit:

```suggestion
Click the green "Use this template" button on this page to create a copy of this repository. This is distinct from a fork in that it can be copied into a private organization and it does not have an upstream link back to this repository.

If you don't use GitHub as your version control system, create a repository in your version control system, clone this repository, add a remote pointing to the repository in your version control system, then push the code into your version control.
```

Reply:
Makes sense -- will update